docs/tools/cli/byoc.md

```diff
@@ -0,0 +1,127 @@
+| ------------------- | -------- | --------------------------------------------------------------------------------------------------------------------- |
+| `--organization-id` | Yes      | Identifier of an organization where to modify a custom cloud                                                          |
+| `--byoc-id`         | Yes      | Identifier of a custom cloud to be modified                                                                           |
+| `--aws-iam-role-arn`| No       | Identifier of the role created when running the infrastructure template in your AWS account                           |
+
+### `avn byoc update`
+
+Modifies a custom cloud configuration.
+
+| Parameter           | Required | Information                                                                                                           |
+| ------------------- | -------- | --------------------------------------------------------------------------------------------------------------------- |
+| `--organization-id` | Yes      | Identifier of an organization where to modify a custom cloud                                                          |
+| `--byoc-id`         | Yes      | Identifier of a custom cloud to be modified                                                                           |
+| `--deployment-model`| No       | Private or public network architecture model defining how resources are arranged and connected to your cloud provider |
+| `--cloud-provider`  | No       | Cloud provider running a custom cloud                                                                                 |
+| `--cloud-region`    | No       | Cloud region where a custom cloud runs                                                                                |
+| `--reserved-cidr`   | No       | IP address range of the VPC in your cloud account for Aiven services created in a custom cloud                        |
+| `--display-name`    | No       | Name of a custom cloud                                                                                                |
+
+## Manage custom cloud permissions
+
+### `avn byoc cloud permissions add`
+
+Grants permissions to a custom cloud.
+
+| Parameter           | Required | Information                                                                                                           |
+| ------------------- | -------- | --------------------------------------------------------------------------------------------------------------------- |
+| `--organization-id` | Yes      | Identifier of an organization where a custom cloud is located                                                         |
+| `--byoc-id`         | Yes      | Identifier of a custom cloud for which to grant permissions                                                           |
+| `--account`         | No       | Identifier of your account                                                                                            |
```

Comment:
This is used for giving the identifier of an account (organizational unit), so that the custom cloud is available in all projects belonging to that account. The description does not make that quite clear I think. In other words, this also grants permission to use the cloud like the option below, but with a different scope.

And the same goes for any instances of the `--account` option in other commands' descriptions.